vanoce.php

```diff
@@ -57,7 +57,16 @@
         <?php foreach ($darky as $darek) { ?>
             <li class="list-group-item">
                 <?php echo $darek["popis"];?>
-                <span class="badge badge-secondary"><?php echo $darek["cena"];?> Kč</span>
+                <?php
```

Comment:
jenom k zamysleni - abychom nemuseli porad otevirat a uzavirat PHP kod - co kdybychom dali tuhle cast rovnou za `foreach`?

Reply:
Tohle je jen ukázkový řešení - sami si to mohou napsat jak jim bude milé. Až to budu livecodovat tak to může ukázat obojace.

Reply:
no snazime se jim vtlouct do hlavy, ze logika a zobrazeni je zvlast - tohle je logika (i kdyz zobrazovaci - to bych ted ale nerozdeloval)